crates/epaint/src/text/font.rs

```diff
@@ -434,7 +440,7 @@ impl Font {
 
     /// Can we display this glyph?
     pub fn has_glyph(&mut self, c: char) -> bool {
-        self.glyph_info(c) != self.replacement_glyph // TODO(emilk): this is a false negative if the user asks about the replacement character itself 🤦‍♂️
+        self.glyph_info_no_cache_or_fallback(c).is_some()
```

Comment:
This seemed a nice "bonus" to-do solving, slightly unrelated to the issue at hand; I'm suspecting there's a reason you chose not to do this in the first place?

Reply:
Well, `glyph_info_no_cache_or_fallback` doesn't use the cache, so it is marginally slower.